app/src/main/java/app/revanced/manager/ui/component/bundle/BundlePatchesDialog.kt

```diff
@@ -62,44 +61,204 @@ fun BundlePatchesDialog(
                         )
-                        HorizontalDivider()
+                    }
+                }
+            }
+            if (!patch.options.isNullOrEmpty()) {
+                AnimatedVisibility(visible = expandOptions) {
+                    val options = patch.options
+
+                    Column {
+                        options.forEachIndexed { i, option ->
+                            OutlinedCard(
+                                modifier = Modifier.fillMaxWidth(), colors = CardColors(
+                                    containerColor = Color.Transparent,
+                                    contentColor = MaterialTheme.colorScheme.onSurface,
+                                    disabledContainerColor = Color.Transparent,
+                                    disabledContentColor = MaterialTheme.colorScheme.onSurface
+                                ), shape = when {
+                                    i == 0 && options.lastIndex == 0 -> RoundedCornerShape(8.dp)
+                                    i == 0 -> RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp)
+                                    i == options.lastIndex -> RoundedCornerShape(bottomStart = 8.dp, bottomEnd = 8.dp)
+                                    else -> RoundedCornerShape(0.dp)
+                                }
+                            ) {
+                                Column(
+                                    modifier = Modifier.padding(12.dp),
+                                    verticalArrangement = Arrangement.spacedBy(4.dp),
+                                ) {
+                                    Text(
+                                        text = option.title,
+                                        style = MaterialTheme.typography.titleMedium,
+                                        color = MaterialTheme.colorScheme.primary
+                                    )
+                                    Text(
+                                        text = option.description, style = MaterialTheme.typography.bodyMedium
+                                    )
+                                }
+                            }
+                        }
                     }
                 }
             }
         }
     }
 }
+
+@Composable
+fun PatchInfoChip(
+    modifier: Modifier = Modifier, onClick: (() -> Unit)? = null, text: String, icon: @Composable (() -> Unit)? = null
+) {
+    val shape = RoundedCornerShape(8.0.dp)
+    val cardModifier = if (onClick != null) {
+        Modifier
+            .clip(shape)
+            .clickable(onClick = onClick)
+    } else {
+        Modifier
+    }
+
+    OutlinedCard(
+        modifier = modifier.then(cardModifier), colors = CardColors(
+            containerColor = Color.Transparent,
+            contentColor = MaterialTheme.colorScheme.onSurface,
+            disabledContainerColor = Color.Transparent,
+            disabledContentColor = MaterialTheme.colorScheme.onSurface
```

Comment:
Same here

Reply:
These parameters are not optional/nullable